olivertwist/main.py

```diff
@@ -33,8 +34,9 @@
     help="Do/Don't open HTML report in browser. Implies --html",
 )
 def main(input, html=True, browser=False):
+    optional_config = ConfigParser.from_default_path()
```

Comment:
mentioned above

Reply:
load based on file path and return a config object
- it can contain default configuration
- or customised if the file path exists